src/auto/implementor.rs

```diff
@@ -0,0 +1,10 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use Object;
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use std::mem;
+use std::ptr;
```

Comment:
Empty file, not regenerated.
4 others too, so need regen after emptying `src\auto`